.travis.yml

```diff
@@ -1,4 +1,4 @@
-dist: xenial # use Ubuntu 16.04
+dist: focal # use Ubuntu 20.04
```

Comment:
Suggestion -- Delete lines 9-11, which now read:
```yml
# make Travis use its legacy infrastructure
sudo: true

```
The `sudo` switch is deprecated and obsolete, https://config.travis-ci.com/, so these lines are dead code.